src/views/product-landing/components/cards/types.ts

```diff
@@ -1,5 +1,7 @@
+import { ProductBrandColor } from 'components/icon-tile/types'
 interface CardInterface {
-  iconSvg?: string
```

Comment:
Have deprecated the `iconSvg` part of this interface, which at least slightly improves the experience of editing the JSON.